Frontend/src/components/DummyComponent/DummyComponent.jsx

```diff
@@ -1,11 +1,44 @@
 import React from 'react';
+import images from '../../constants/images';
 import styles from './DummyComponent.module.css';
 
 const DummyComponent = () => {
   return (
-    <div className={styles.headingContainer}>
-      <h1 className={styles.heading}>Welcome To BMC</h1>
-      <p className={styles.text}>We are glad to have you</p>
+    <div className={styles.bodyContainer}>
+      <div  className={styles.left}>
+        <div className="text-container">
+          <h1>TEMIBMC</h1>
+          <p>
+            Your business tool used to visualize all the building blocks when you want to start a business, including customers, route to market, value proposition and finance.
+          </p>
+        </div>
+      </div>
+      <div  className={styles.right}>
+        <div  className='form-text'>
+          <h2>Sign In</h2>
+          <button type='button' className='signIn-button'>Sign in with google</button>
+          <p>-------- Or --------</p>
+        </div>
+        <div className='form'>
+          <form action="">
+            <label htmlFor="full-name"> Enter Full Name or Email
+              <input type="text" name="full-name" id="text" />
+            </label>
+            <label htmlFor="full-name"> Enter Password
+              <input type="text" name="full-name" id="text" />
+            </label>
+            <div  className='checkboxes'>            
+              <input type="checkbox" name="" id="" />Password must have at least 8 characters <br />
```

Comment:
This strict validation is not needed for now. Let's make it easy for them to create passwords - remove the upper case and 1 digit thing - leave character count

Reply:
oh, it's just character count and empty input validation at the moment